composer.json

```diff
@@ -23,7 +23,9 @@
         "doctrine/cache": "^1.10",
         "symfony/routing": "^5.1",
         "symfony/http-foundation": "^5.1",
-        "symfony/http-kernel": "^5.1"
+        "symfony/http-kernel": "^5.1",
+        "tamtamchik/simple-flash": "^2.0",
+        "caxy/php-htmldiff": "^0.1.12"
```

Comment:
Ah au fait, même si elle est super pratique, la librairie caxy/php-htmldiff  pèse presque 9 Mo, ... est-ce un poids justifié pour la fonctionnalité et son usage ? @mrflos qu'en dis-tu ?

Reply:
J'ai commenté la PR sur https://github.com/caxy/php-htmldiff/pull/86 et ya eu du retour!
Si la dépendance ne fait plus qu'1 mo, c'est raisonnable, 10mo...  ...ca se réfléchit..

Reply:
ah ah lol j'avais oublié ce détail !! en fait c'est juste à cause de 1 fichier `vendor/caxy/php-htmldiff/demo/tablediffs.json`, y'a moyen de le supprimer avant de créer le .zip ?

Reply:
ah effectivement, en attendant la PR sur caxy, nous pourrions utiliser un script composer de post-instal : https://getcomposer.org/doc/articles/scripts.md#what-is-a-script-

qui retire le fichier en trop

Reply:
bonne idée, c'est fait !

Reply:
pour info, la version v0.1.13 permet d'éviter le chargement du dossier demo, grace a un fichier .gitattributes donc pas besoin du script de nettoyage cf. https://github.com/caxy/php-htmldiff/pull/86

Reply:
Ok merci :)